src/components/modals/WelcomeModal.tsx

```diff
@@ -113,6 +113,7 @@ const WelcomeModal: FC<IModal> = ({ setShowModal }) => {
 };
 
 const ModalGrid = styled.div`
+	color: black;
```

Comment:
@mateodaza please use our design system colors, I think it's neutralColor.gray[100]